scenario.rb

```diff
@@ -0,0 +1,129 @@
+#users
+#boards
+#lists
+#cards
+#comments
+#Create 10 any unit tests for this classes and their methods
+
+require_relative "lists"
+require_relative "comments"
+require_relative "cards"
+require_relative "boards"
+require_relative "team"
+require_relative "users"
+
+
+
+#create new user
+user = Users.new
+user.users << user
```

Comment:
don't you thinks it looks strange for user  to "know" about other users
I suppose users should be stored in team or board at least